pom.xml

```diff
@@ -613,6 +625,106 @@ under the License.
 
 	<profiles>
 
+		<profile>
+			<id>scala-2.11</id>
```

Comment:
missing activation properties; in the current form one always has to explicitly activate one scala profile or the build will simply fail.

Reply:
See comment above.